api/models.py

```diff
@@ -28,3 +31,49 @@ def __init__(self, username: str, email: str, password: str):
 
     def __repr__(self):
         return f'<User {self.username}>'
+
+
+# Companies Models Class:
+class CompaniesModel(db.Model):
+    __tablename__ = 'companies'
+
+    # Table attributes:
+    company_id = db.Column(db.Integer, primary_key=True)
+    company_name = db.Column(db.String(MAX_CREDENTIAL_LEN))
+    company_info = db.Column(db.Text)
+    num_listings = db.Column(db.Integer)
+
+    def __init__(self, c_id: int, c_name: str, num_listings: int):
+        self.c_id = c_id
+        self.c_name = c_name
+        self.num_listings = num_listings
+
+    def __init__(self):
+        return f'<Company {self.c_name}>'
+
+
+# Listings Models Class:
+class ListingsModel(db.Model):
+    __tablename__ = 'listings'
+
+    # Table attributes:
+    id = db.Column(db.Integer, primary_key=True)
+    company_id = db.Column(db.Integer, db.ForeignKey('companies.company_id'))
+    position = db.Column(db.String(MAX_CREDENTIAL_LEN))
+    pos_responsibility = db.Column(db.Text)
+    min_qualifictions = db.Column(db.Text)
+    pref_qualifictions = db.Column(db.Text)
+    additional_info = db.Column(db.Text)
+    
+    def __init__(self, company_id: int, position: str, pos_responsibility: str,
+                min_qualifictions: str, pref_qualifictions: str, 
+                additional_info: str):
+        self.company_id = company_id
+        self.position = position
+        self.pos_responsibility = pos_responsibility
+        self.min_qualifictions = min_qualifictions
+        self.pref_qualifictions = pref_qualifictions
+        self.additional_info = additional_info
```

Comment:
In ListingsModel min_qualifictions and pref_qualifictions should be min_qualifications and pref_qualifications.